packages/deployer/src/utils/validation.ts

```diff
@@ -0,0 +1,13 @@
+import { Config } from '@api3/node';
+import * as logger from '../utils/logger';
+
+export function validateConfig(config: Config, nodeVersion: string) {
+  if (nodeVersion !== config.nodeSettings.nodeVersion) {
```

Comment:
You've dropped the `cloudProvider` check in the validation?

Reply:
Yes I did, it will be done by validator anyway.